src/netconfig.cpp

```diff
@@ -379,7 +381,7 @@ static const Command ifconfigCommands[] = {
     {"reset", nullptr, "Reset configuration to factory defaults", cmdReset},
     {"mac", "{INTERFACE} MAC", "Set MAC address", cmdMac},
     {"hostname", "NAME", "Set host name", cmdHostname},
-    {"gateway", "IP", "Set default gateway", cmdGateway},
+    {"gateway", "IP", "Set default gateway", cmdGateway, "NOTE: The system doesn't allow unreachable gateways. Please ensure that your gateway \nbelongs to the network defined by the IP address and the Netmask."},
```

Comment:
This message doesn't fit into 80 characters. I suggest to move `gateway` at the next line.
It will look like:
```
NOTE: The system doesn't allow unreachable gateways. Please ensure that your
gateway belongs to the network defined by the IP address and the netmask.
```
Capitalized `Netmask` seems to me strange. I would prefer to write it in lower case.